include/crm/pengine/internal.h

```diff
@@ -587,6 +587,8 @@ bool pe__uname_has_tag(pe_working_set_t *data_set, const char *node, const char
 
 bool pe__rsc_running_on_any_node_in_list(pe_resource_t *rsc, GList *node_list);
 GList *pe__filter_rsc_list(GList *rscs, GList *filter);
+GList * pe__build_uname_list(pe_working_set_t *data_set, const char *s);
```

Comment:
uname is actually a misnomer -- a long time ago, a node name had to be the same as the host's uname, but now they can be different, so let's use node_name or something like that instead

Reply:
Fixed, though there's still variables named "unames" here and there.  I could also change that either as part of this PR or separately.

Reply:
No need, we can change them as we need to modify code that uses them. Unfortunately some of them are public API, too.